packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -361,13 +381,15 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           // up-to-date version of this module.
           try {
             // delay setting `isSelfAccepting` until the file is actually used (#7870)
-            const depModule = await moduleGraph.ensureEntryFromUrl(
-              unwrapId(url),
-              ssr,
-              canSkipImportAnalysis(url)
+            const depModule = moduleGraph.ensureEntryFromResolved(
```

Comment:
Replaced `ensureEntryFromUrl` call with `ensureEntryFromResolved` to avoid the extra `resolveUrl` call